src/test/java/seedu/address/storage/JsonAdapterOrderTest.java

```diff
@@ -0,0 +1,137 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.storage.JsonAdaptedOrder.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalOrders.ORDER;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.Date;
+import seedu.address.model.Label;
+import seedu.address.model.order.Amount;
+import seedu.address.model.order.Customer;
+
+
+public class JsonAdapterOrderTest {
+    private static final String INVALID_LABEL = "TAS$%$#";
+    private static final String INVALID_DATE = "28@nextmonth";
```

Comment:
Good work! Test cases look fairly comprehensive; appreciate the effort you went through to come up with all the examples of incorrect json files.

Reply:
Thanks!!